### ISSUE TITLE
Disk Utility not seeing upgraded SSD

### ISSUE THREAD

**Issue by jmarsil** [MisterFMagic/Yoga-720-opencore#2]
Hey! I replaced the native SSD with a 1TB upgrade from crucial (CT1000P5PSSD8), however, I am still unable to see the it in disk utility when I load the installer from USB. Any potential fixes?

**Comment by MisterFMagic** [comment]
Hey! Please check if SATA mode is correctly set to AHCI in the BIOS/UEFI settings

**Comment by jmarsil** [comment]
Yes it's enabled. Here are my Bios settings, lmk if you see anything that looks off?

![image](https://user-images.githubusercontent.com/39134049/162211033-2630da5b-182e-44c9-935c-ac81e8be7124.jpeg)

![image](https://user-images.githubusercontent.com/39134049/162211048-229ce0fa-1add-4c5f-bc86-3dc59f9c2a82.jpeg)

![image](https://user-images.githubusercontent.com/39134049/162211072-2531581f-f344-4bd0-9ae1-11227419ca60.jpeg)

![image](https://user-images.githubusercontent.com/39134049/162211106-24c4fd35-1f52-4778-8224-4f104dc49860.jpeg)

**Comment by jmarsil** [comment]
Managed to get to the disk utility screen again after sometime, still doesn't see ssd

![image](https://user-images.githubusercontent.com/39134049/162234910-d32f3b0a-a022-4811-9be6-f95b2d15d8d2.jpeg)

**Comment by MisterFMagic** [comment]
Hi! Sorry for the late reply. Your settings are looking fine to me, although "product name", "MTM" and "Lenovo sn" are "INVALID" on the information screen. That's at least a bit strange. Is this the latest firmware for your device?

I would try to add "NVMeFix" Kext https://github.com/acidanthera/NVMeFix to the config. Good luck!
